concepts/80 Troubleshooting/50 A Drop-Down Editor Does Not Show Data/00 A Drop-Down Editor Does Not Show Data.md

```diff
@@ -0,0 +1,9 @@
+This section describes the issues related to the following drop-down editors:
+
+- [SelectBox](/Documentation/Guide/UI_Components/SelectBox/Getting_Started_with_SelectBox/)
+- [Lookup](/Documentation/Guide/UI_Components/Lookup/Getting_Started_with_Lookup/)
+- [DropDownBox](/Documentation/Guide/UI_Components/DropDownBox/Getting_Started_with_DropDownBox/)
+- [TagBox](/Documentation/Guide/UI_Components/TagBox/Getting_Started_with_TagBox/)
+- [DataGrid](/Documentation/Guide/UI_Components/DataGrid/Getting_Started_with_DataGrid/)/[TreeList](/Documentation/Guide/UI_Components/TreeList/Getting_Started_with_TreeList/) lookup column editor
```

Comment:
DataGrid and TreeList cannot be named as drop-down editors. I think it's more correct to write like 
[DataGrid|lookup](link_here)